src/UI/Art/SpriteArtist/UnitSpriteSet.java

```diff
@@ -296,7 +296,7 @@ public void drawUnitIcons(Graphics g, Commander[] COs, Unit u, int animIndex, in
     if( !statusIcons.isEmpty() )
     {
       int iconIndex = (animIndex%(statusIcons.size()*ANIM_FRAMES_PER_MARK))/ANIM_FRAMES_PER_MARK;
-      BufferedImage statusIcon = statusIcons.get(iconIndex);
+      BufferedImage statusIcon = statusIcons.get(Math.abs(iconIndex));
```

Comment:
Wasn't this issue fixed elsewhere already?

Reply:
It was... but not on this branch, and I ran into it during testing. Guess I accidentally checked it in.